### PR TITLE
Navigation component done.

### PR DESCRIPTION
**Pull request by TJain02**
Screenshot (120)

### CODE REVIEW

File: CSS Components/navigation.css [hunk 1]
```diff
@@ -0,0 +1,22 @@
+
+.navigation-container{
+    font-size: 1.25rem;
+    display: flex;
+    justify-content: space-between;
+    background-color:#090553;
+    padding:1rem;
+    color: #eef2ff;
```
Comment (arulk8) [inline CSS Components/navigation.css]: move this to variable file if you already have

**Review by sashaboi — APPROVED** [review]
Looking good ! good to merge

**Review by arulk8 — APPROVED** [review]
Good for me. After writing navbar many time I can visualize your navbar without preview. hehe.

File: navigation.html [hunk 2]
```diff
@@ -0,0 +1,26 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="utf-8">
+    <meta name="viewport" content="width=device-width ">
+    <title>Navigation</title>
+    <link rel="stylesheet" href="/CSS Components/navigation.css">
```
Comment (shubhmehra) [inline navigation.html:7]: avoid using spaces in file names

**Pull request merged**